Software/Python/easygopigo.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import division
+from builtins import input
+
+import sys
+import tty
+import select
+import gopigo
+
+sys.path.insert(0, '/home/pi/Desktop/GoPiGo/Software/Python/line_follower')
```

Comment:
Yes. This can go as soon as the line_sensor library is system-wide.
Also, can you look into moving functions from scratch_line.py into line_sensor.py ? 
I think a lot of people will only need to get a yes or no from each sensor, instead of doing the calculations themselves. And those who need a yes/no may not have the know-how to figure out that the code is in scratch_line.py